plugin/src/test/java/io/spine/tools/gradle/bootstrap/func/SpineBootstrapPluginTest.java

```diff
@@ -50,7 +51,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(TempDirectory.class)
-@Functional
+@SlowTest
 @DisplayName("`io.spine.bootstrap` plugin should")
```

Comment:
I think the plugin is now called `io.spine.tools.gradle.bootstrap`. Could you please fix the test name?